proposals/0069-swift-mutability-for-foundation.md

```diff
@@ -220,8 +220,6 @@ URLComponents | NSURLComponents
 URLQueryItem | NSURLQueryItem
 UUID | NSUUID
 
-These new `struct` types will be implemented in the Swift overlay. Immutable/mutable pairs (e.g. `Data` and `MutableData`) will become one mutable struct type.
```

Comment:
I removed this sentence, because a very similar sentence appears right before the table.